test/models/git/repository_test.rb

```diff
@@ -0,0 +1,69 @@
+require 'test_helper'
```

Comment:
This file was named incorrectly (`repository.rb` instead of `repository_test.rb`).

Reply:
Good catch. It won't have been been called during CI.

Reply:
I couldn't find the tests, which I found very odd. But this was the reason :)